tool_packages/unique_web_search/tests/test_search_engines.py

```diff
@@ -78,6 +94,38 @@ def test_tavily_config_creation(self):
         assert hasattr(config, "fetch_size")
         assert hasattr(config, "custom_search_config")
 
+    def test_vertexai_config_creation(self):
+        """Test VertexAIConfig creation."""
+        config = VertexAIConfig(search_engine_name=SearchEngineType.VERTEXAI)
+
+        assert config.search_engine_name == SearchEngineType.VERTEXAI
+        assert hasattr(config, "model_name")
+        assert hasattr(config, "grounding_system_instruction")
+        assert hasattr(config, "requires_scraping")
+        assert hasattr(config, "enable_entreprise_search")
+        assert hasattr(config, "enable_redirect_resolution")
+        assert config.model_name == "gemini-2.5-flash"  # default value
+        assert not config.requires_scraping  # default value
+        assert not config.enable_entreprise_search  # default value
+        assert config.enable_redirect_resolution  # default value
```

Comment:
remove all the `#default value` and rename the test to `test_vertexai_config_creation_default_value` instead